marketplace-and-integrations/integrations/algolia.md

```diff
@@ -171,6 +171,23 @@ services.AddUmbraco(_env, _config)
     .Build();
 ```
 
+### Version 2.1.5 and up
```

Comment:
move this section above ### Version 2.0.0 to 2.1.5, as the newest are documented first. So the order would be: Version 2.0.0 to 2.1.5 **then** Version 2.0.0 to 2.1.5 **then** Up to version 1.5.0